nearcore/src/runtime/mod.rs

```diff
@@ -2728,11 +2766,7 @@ mod test {
             env.step(vec![vec![], vec![]], vec![true, true], ChallengesResult::default());
         }
         // which validator is kicked out depends on which validator selection is used
```

Comment:
fix the comment

Reply:
This comment was supposed to be in completely another place oO